docs/DXIL.rst

```diff
@@ -88,21 +88,29 @@ The shader model is specified as a named metadata in DXIL::
   !dx.shaderModel = !{ !0 }
   !0 = !{ !"<shadelModelName>", i32 <major>, i32 <minor> }
 
-The following values of <shaderModelName>_<major>_<minor> are supported:
-
-====================      ===================================== ===========
-Target                    Legacy Models                         DXIL Models
-====================      ===================================== ===========
-Vertex shader (VS)        vs_4_0, vs_4_1, vs_5_0, vs_5_1        vs_6_0
-Hull shader (HS)          hs_5_0, hs_5_1                        hs_6_0
-Domain shader (DS)        ds_5_0, ds_5_1                        ds_6_0
-Geometry shader (GS)      gs_4_0, gs_4_1, gs_5_0, gs_5_1        gs_6_0
-Pixel shader (PS)         ps_4_0, ps_4_1, ps_5_0, ps_5_1        ps_6_0
-Compute shader (CS)       cs_5_0 (cs_4_0 is mapped onto cs_5_0) cs_6_0
-Shader library            no support                            lib_6_1
-Mesh shader (MS)          no support                            ms_6_5
-Amplification shader (AS) no support                            as_6_5
-========================= ===================================== ===========
+The following values of ``<shaderModelName>``, ``<major>``, ``<minor>`` are supported:
```

Comment:
Thanks for noticing and updating this!

I think the formatting with `_` between these was meant to match the values in the table below, **however this section is quite out of date anyway**.

Here are some problems with the table as it stands:
- We don't actually support these "Legacy Models" in DXIL.  When this was originally written, we did, but later decided to upgrade the shader model in dxc.exe, and not support legacy values at the API, nor capture them into the DXIL.
- The "DXIL Models" column only lists the very first shader model supported for various shader stages.
- lib_6_1 was added here before we officially supported libraries as a target the runtime would accept at lib_6_3, so while you can technically compile to lib_6_1 and lib_6_2, you must disable validation to do so.

Given all of this, I feel like we should do away with this specific table format.  Perhaps we should change columns to something like this:
```
| Shader Kind | shaderModelName | Minimum major, minor |
+=======================+=======================================+=============+
| Vertex shader (VS)    | vs | 6, 0 |
...
| Dxil library | lib | 6, 3 |
```
... and so on ...
(note: corrected library line here)

What do you think?

Reply:
Thanks @tex3d, for the suggestions regarding the correctness of the content. Happy to update the content.

Are Mesh Shader (MS) and Amplification Shader (AS) supported? If not, do they need to be listed as `unsupported` (for completeness??)?